plugins/conversation-contracts/mcp/main.go

```
@@ -0,0 +1,477 @@
+
+	// Ensure parent directory exists.
+	dir := filepath.Dir(jsonlPath)
+	if err := os.MkdirAll(dir, 0o755); err != nil {
+		return "", fmt.Errorf("mkdir %s: %w", dir, err)
+	}
+
+	f, err := os.OpenFile(jsonlPath, os.O_APPEND|os.O_CREATE|os.O_WRONLY, 0o644)
+	if err != nil {
+		return "", fmt.Errorf("open jsonl %s: %w", jsonlPath, err)
+	}
+	defer f.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `f.Close` is not checked

(errcheck)